src/utils/symbol-cache.ts

```
@@ -0,0 +1,75 @@
+import { Node, Symbol, TypeChecker } from 'ts-morph';
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)